src/main/java/pl/cleankod/exchange/core/trace/WebMvcConfig.java

```diff
@@ -0,0 +1,20 @@
+package pl.cleankod.exchange.core.trace;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebMvcConfig implements WebMvcConfigurer {
```

Comment:
Same question as above, why is this config class in the `core` package of the application.